paths/paths.go

```diff
@@ -55,7 +55,16 @@ func DefaultTailscaledStateFile() string {
 		return f()
 	}
 	if runtime.GOOS == "windows" {
-		return filepath.Join(os.Getenv("LocalAppData"), "Tailscale", "server-state.conf")
+		programData := filepath.Join(os.Getenv("ProgramData"), "Tailscale", "server-state.conf")
+		if _, err := os.Stat(programData); err == nil {
+			return programData
+		}
+		localAppData := filepath.Join(os.Getenv("LocalAppData"), "Tailscale", "server-state.conf")
```

Comment:
// This is where Tailscale 1.14 and earlier stored the server-state. (Issue NNN)
// We still recognize it as a fallback.

Reply:
Done.